src/index.js

```diff
@@ -1,20 +1,32 @@
 import './style.css';
 
-const todosList = [
-  { description: 'hello', completed: false, index: 1 },
-  { description: 'done', completed: false, index: 2 },
-];
+import render from './populateui.js';
+import AddToDoList from './list.js';
 
-const render = (list) => {
-  const sortedTodos = list.sort((a, b) => a.index - b.index);
-  const todosContainer = document.querySelector('.lists');
-  let todosHtml = '';
-  sortedTodos.forEach((todo) => {
-    todosHtml += ` <div class="todo-item">
-        <input type="checkbox" /><span> ${todo.description}</span>
-    </div>`;
-  });
-  todosContainer.innerHTML = todosHtml;
-};
+const taskList = new AddToDoList();
+render(taskList);
 
-render(todosList);
+const addTodoBtn = document.querySelector('.add-btn');
+addTodoBtn.addEventListener('click', () => {
+  const id = `id${Math.random().toString(16).slice(2)}`;
+  const description = document.querySelector('.input-list').value.trim();
+  const completed = false;
+  const index = taskList.list.length + 1;
+
+  const newTask = {
+    id,
+    description,
+    completed,
+    index,
+  };
+  if (description) {
+    taskList.addTask(newTask);
+    render(taskList);
+  }
+});
+
+const clearBtn = document.querySelector('.clear-btn');
+clearBtn.addEventListener('click', () => {
+  taskList.clearCompletedTask();
+  render(taskList);
+});
```

Comment:
- [ ] I noticed that when the user needs to edit a task is not possible, you need to add a function to edit task then the user will be able to edit a task and if the user refreshes the page the task edited will be save in the local storage 🙃 🙃 , please kindly fix it 😁 😁